core/lib/vm/src/tracers/traits.rs

```diff
@@ -88,10 +81,22 @@ pub enum TracerExecutionStatus {
     Stop(TracerExecutionStopReason),
```

Comment:
Would using a `Result<(), TracerExecutionStopReason>` instead of `TracerExecutionStatus` be non-idiomatic? Or is it the case that other statuses besides `Continue` and `Stop` could be added in the future?

Reply:
I don't feel this right.. 
Stop is not an `Error` and `Continue` is not Ok..